tests/app/service/test_rest.py

```diff
@@ -720,18 +737,20 @@ def test_cant_update_service_org_type_to_random_value(client, sample_service):
     auth_header = create_admin_authorization_header()
 
     resp = client.post(
-        "/service/{}".format(sample_service.id),
+        f"/service/{sample_service.id}",
         data=json.dumps(data),
         headers=[("Content-Type", "application/json"), auth_header],
     )
-    assert resp.status_code == 500
+    assert resp.status_code == 400
```

Comment:
This was checking for a `500` previously?  Ooof...

Reply:
Yeah, it was. But it looked like 400 is correct. I spent ~ a full day or so double and triple checking this, as it was one of those last really hard to identify 3 tests that were failing.